src/parser/MaintainSentences.cpp

```diff
@@ -254,9 +260,15 @@ std::string CreateEdgeIndexSentence::toString() const {
     buf += " ON ";
     buf += *edgeName_;
     buf += " (";
-    std::string columns;
-    folly::join(", ", this->columns(), columns);
-    buf += columns;
+    std::vector<std::string> fieldDefs;
+    for (const auto& field : this->fields()) {
```

Comment:
I think you could move these code to `IndexFieldList::toString()`

Reply:
> I think you could move these code to `IndexFieldList::toString()`

Yeah, I agree with your suggestion, this PR will solve the urgent compilation problem, which I think I will deal with later. thanks for your careful reading.